vercel.json

```diff
@@ -570,7 +570,7 @@
     },
     {
       "source": "/test-suites/_archive/ci-integration/circle-ci",
-      "destination": "/reference/ci-workflows/recording-strategies"
+      "destination": "/reference/ci-workflows/upload-strategies"
```

Comment:
Yes. I think describing *recording* strategies raises the complexity here and we might not actually need this (see [here](https://github.com/replayio/docs/pull/202#discussion_r1636690390)). Since this content did exist, I had to redirect it somewhere and `/upload-strategies` seemed like a good fit